cli/src/test-utils/test-logger.ts

```diff
@@ -0,0 +1,36 @@
+import chalk from "chalk";
+
+const TestLogger = {
+  error,
+  log,
+  mute,
+  success,
+  warn
+};
+
+export default TestLogger;
```

Comment:
I would personally recommend not using default exports (using a named import instead with `export const TestLogger`), to ensure that it's called the same in the file that defines it and it files that import it.